docs/sources/write/grot-guides/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Grot guides
+review_date: "2024-08-29"
+description: Understand what Grot guides are and how to use them in your Markdown.
+keywords:
+  - Hugo
+  - shortcodes
+weight: 501
+---
+
+# Grot guides
+
+The `guide` shortcode embeds a Grot guide into the docs that people can follow to get help and guidance on topics and direction to documentation resources.
+
+```markdown
+{{</* guide name="instrument" title="title" text="text" button="button" */>}}
+```
+
+The website looks up guide Markdown files from the website repository in the `content/guides` folder. The shortcode accepts the following parameters:
+
+| Parameter | Description                                                                 | Required |
+| --------- | --------------------------------------------------------------------------- | -------- |
+| `name`    | The name of the guide Markdown file in the website `content/guides` folder. | yes      |
+| `title`   | A custom guide title to override the default guide title.                   | no       |
+| `text`    | Custom body text to override the default guide text.                        | no       |
+| `button`  | Custom button copy to override the default guide button copy.               | no       |
+
+If you don't set custom title, text, and button values, the guide introduction screen uses the following default text:
+
+![Grot guide introduction screen with default copy](https://grafana.com/media/docs/writers-toolkit/grot-guide.png)
+
+Guide content consists of various YAML structures in Markdown front matter for the guide meta, header, welcome, and screens.
+
+## Meta
+
+Customize the `name`, `api`, and `title` fields, and leave the rest as is:
+
+```yaml
+# name that matches the filename and name used to embed
+name: instrument
+# match the name field
+api: instrument
+title: Applicating Instrumentation
+type: guides
+layout: single
+# Hugo rendering options
+_build:
+  render: false
+  list: true
+```
+
+## Header
+
+The header structure defines the guide's image and image spacing:
+
+```yaml
+header:
+  image:
+    src: /media/guides/grafana-guides-whichgrafana-header.svg
+    alt: Grot metrics wizard
+    width: 221
+    height: 131
+```
```

Comment:
I think if this is an example, rather than a copyable template we should say that.
Perhaps something like:

````suggestion
The header structure defines the guide's image and image spacing.
The following example use the image at the URL `/media/guides/grafana-guides-whichgrafana-header.svg`, specifying the alternative text `Grot metrics wizard`, and setting a width of `221` px and height of `131` px.:

```yaml
header:
  image:
    src: /media/guides/grafana-guides-whichgrafana-header.svg
    alt: Grot metrics wizard
    width: 221
    height: 131
```
````